test/config/spiffe-csi-driver.yaml

```diff
@@ -50,7 +50,7 @@ spec:
             # driver will mount this directory into containers.
             - mountPath: /spire-agent-socket
               name: spire-agent-socket-dir
-              readOnly: true
+                #readOnly: true
```

Comment:
```suggestion
```

Or we could explicitly set it to false, with a comment about why?

Reply:
Oh whoops, I didn't mean to leave this commented line in. But yes, I agree that being explicit and adding a comment would be very helpful and self-documenting.